kola/tests/ostree/basic.go

```diff
@@ -195,7 +195,7 @@ func ostreeRemoteTest(c cluster.TestCluster) {
 
 		osRemoteListSplit := strings.Split(string(osRemoteListOut), "\n")
 		// should have original remote + newly added remote
-		if len(osRemoteListSplit) != 2 {
+		if len(osRemoteListSplit) != 1 {
```

Comment:
The above comment seems to indicate that there should also be the original remote in addition to the newly added remote. Also, is this tested against FCOS?

cc @miabbott

Reply:
No it's not tested against FCOS  .... good call.

Reply:
RHCOS has no remote nowadays, but FCOS likely will.

FCOS tests are going to be a non-starter right now #981 

You should still be able to inspect what the state of FCOS is like with `coreos-assembler run`, though.

Reply:
I'm downloading a copy right now to inspect.

Reply:
We'll probably have to split the test then to accommodate both distros, e.x.: `rhcos.ostree.remote` which calls a RHCOS specific function that calls `ostreeRemoteTest` passing in the expected amount of remotes existing in the base image.

Reply:
My locally built FCOS shows:

```
$ rpm-ostree status
State: idle
Warning: failed to query journal: Cannot assign requested address (os error 99)
AutomaticUpdates: disabled
Deployments:
● ostree://fedora-coreos:fedora/29/x86_64/coreos
                   Version: 29 (2019-02-28T22:02:19Z)
                    Commit: d18e5962c1771c0eb3afccf694d61cce1ebafb26ae50104dee9e3284cbce2b17
```